src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -59,6 +60,21 @@ public void setElevatorPosition(Measure<Angle> position) {
     elevatorMotor.setControl(positionRequest.withPosition(position.in(Units.Rotations)));
   }
 
+  /**
+   * @return The current position of the elevator in rotations
+   */
+  public double getElevatorPosition() {
+    return elevatorMotor.getPosition().getValueAsDouble();
+  }
+
+  /**
+   * @return If the elevator position is within tolerance of desired position
+   */
+  public boolean isElevatorAtPosition(Measure<Angle> position) {
+    return (Math.abs(getElevatorPosition() - position.in(Units.Rotations)) < constElevator.AT_POSITION_TOLERANCE
```

Comment:
After getElevatorPosition is returned as a rotations measure
```suggestion
    return (Math.abs(getElevatorPosition().minus(position).in(Units.Rotations)) < constElevator.AT_POSITION_TOLERANCE
```